phoenix-core/src/main/java/org/apache/phoenix/jdbc/PhoenixStatement.java

```diff
@@ -1578,6 +1580,12 @@ public ExplainPlan getExplainPlan() throws SQLException {
 
                 @Override
                 public MutationState execute() throws SQLException {
+                    if (CDCUtil.isACDCIndex(
+                            ExecutableDropIndexStatement.this.getIndexName().getName())) {
+                        throw new SQLExceptionInfo.Builder(CANNOT_DROP_CDC_INDEX).setTableName(
+                                ExecutableDropIndexStatement.this.getIndexName().getName())
```

Comment:
Nit: `ExecutableDropIndexStatement.this.getIndexName().getName()` could be assigned to a local variable to avoid repetition.

Reply:
@haridsv , updated